app/backend/approaches/chatreadretrievereadvision.py

```diff
@@ -117,7 +121,7 @@ async def run_until_final_call(
         )
 
         chat_completion: ChatCompletion = await self.openai_client.chat.completions.create(
-            model=self.gpt4v_deployment if self.gpt4v_deployment else self.gpt4v_model,
+            model=self.chatgpt_deployment if self.chatgpt_deployment else self.chatgpt_model,
```

Comment:
Not a strictly needed change, but something I realized a few weeks ago: We shouldn't need to use a vision model for the query rewrite phase. We don't need the capability, and the vision models are slower.
I can also make this in separate PR if desired.

Reply:
Made separate PR- https://github.com/Azure-Samples/azure-search-openai-demo/pull/1659